core-services/release-controller/_releases/release-okd-scos-4.y-stable.json

```diff
@@ -2,7 +2,6 @@
   "name":"4-scos-stable",
   "as": "Stable",
   "message": "All amd64 OKD/SCOS versions are tracked here",
-  "overrideCLIImage": "image-registry.openshift-image-registry.svc:5000/ocp/4.6:cli",
```

Comment:
Is there any specific reason that you removed this?
This is a release-controller specific setting that should remain in place.

Reply:
This aligns this config with all the other OKD releases configs - only this one currently has the override.
Are we sure this is still required? 4.6 is EOL

Reply:
The setting has no real bearing on anything but how the release-controller creates it's releases.  Ultimately, I think it's safe to remove, but I wanted to make sure there wasn't a specific reason that we needed to be aware of.